docs/source/modules/submission.rst

```diff
@@ -1,76 +1,75 @@
-This enables us to score new models on existing benchmarks,
-as well as new benchmarks against existing models.
+To submit a model or benchmark to the Brain-Score leaderboard, Brain-Score provides an automated submission system.
+This allows us to publicly score new models on existing benchmarks and score existing models against new benchmarks.
 
 .. image:: docs/source/modules/brainscore_submission.png
-    :width: 200px
+    :width: 400px
     :align: center
-    :height: 100px
+    :height: 200px
     :alt: submission system diagram
 
-- **Brain-Score Website**:
+Users can submit models or benchmarks as a zip file via the `Brain-Score website <#Brain-Score Website>`_ or as a pull request via GitHub.
+Appropriately configured zip file submissions are converted by `Jenkins <#Jenkins>`_ (`create_github_pr`) into a pull request. 
+If tests pass, the PR is automatically merged into the codebase by a GitHub Action (`automerge_plugin-only_prs`).
+If tests fail on a website submission, the submitter will be notified via email.
+Note: While we currently do not support private code submissions, this feature is under development.
+
+If a newly merged PR contains new models or benchmarks,
+a separate Github Action (`score_new_plugins`) triggers the scoring runs which are handled by Jenkins
+and run on the `OpenMind compute cluster <#Openmind>`_.
 
-    The `website <www.brain-score.org>`_ (`Github <https://github.com/brain-score/brain-score.web>`_) handles our front
-    end. It is implemented using Django  and also accesses the database instance.
 
-    Users can submit plugins as a zip file via the website or as a pull request via Github.
-    Zip file submissions are converted into a pull request that is automatically merged if tests pass.
-    Once a pull request has been merged and if there are new model or benchmark plugins,
-    a Github action triggers the scoring runs which are handled by Jenkins
-    and run on the OpenMind compute cluster.
+Submission System Components
+############################
 
-    The website is hosted via Amazon Elastic Beanstalk. There are three insteances on AWS EB:
+- **Brain-Score Website**:
+
+    The `website <www.brain-score.org>`_ (`GitHub <https://github.com/brain-score/brain-score.web>`_) handles our front 
+    end. It is implemented using Django and also accesses the database instance.
+
+    The website is hosted via Amazon Elastic Beanstalk. There are three instances on AWS EB:
 
     - `brain-score-web-dev <brain-score-web-dev.us-east-2.elasticbeanstalk.com>`_: This is our dev website environment.
       It is the same as prod, but is a retired branch.
-
     - `brain-score-web-prod <brain-score-web-prod.us-east-2.elasticbeanstalk.com>`_: Our old (retired) logacy branch
       of prod. This was used from 6/2020 - 6/2023, and was replaced by the updated environment below.
-
     - `Brain-score-web-prod-updated <http://brain-score-web-prod-updated.kmk2mcntkw.us-east-2.elasticbeanstalk.com>`_:
```

Comment:
We can remove the updated env, it is no longer needed. Also lines 34-37 would become: 

- `brain-score-web-dev <brain-score-web-dev.us-east-2.elasticbeanstalk.com>`_: This is our dev website environment.  It is the same as prod but used the dev database.

- `brain-score-web-prod <brain-score-web-prod.us-east-2.elasticbeanstalk.com>`_: Our production branch
       of prod, nearly identical copy to dev.